build/Dockerfile

```diff
@@ -19,10 +19,8 @@ COPY . .
 
 RUN make build
 
-FROM alpine:3.15.8
+FROM gcr.io/distroless/static-debian11:nonroot AS backup-restore
 
-RUN apk add --update bash curl
-
-COPY --from=builder /go/src/github.com/gardener/backup-restore/bin/etcdbrctl /usr/local/bin/etcdbrctl
+COPY --from=builder /go/src/github.com/gardener/backup-restore/bin/etcdbrctl /etcdbrctl
```

Comment:
Is there any particular reason to copy `etcdbrctl` binary to `/` dir and not `/usr/local/bin`?